position/async/publishPosition.js

```diff
@@ -1,11 +0,0 @@
-const buildPosition = require('../async/buildPosition')
```

Comment:
I think this is a totally unused method.
It's using the generic `buildPosition` which is mainly used by higher level methods like `buildChooseOnePosition`